spotbugs/etc/messages.xml

```diff
@@ -8288,10 +8288,11 @@ String constructComponentName() {
           (and hopefully better) static analysis technique.
           We are interested is getting feedback about the
           usefulness of this bug pattern.
-          To send feedback:
+          To send feedback, either:
```

Comment:
"To send feedback, either:" -> "For sending feedback, check:"

Reply:
OK.